### PR TITLE
shijiay result

### PR DESCRIPTION
**Pull request by bronyayang**
Val Loss: 3.87

Link: https://api.wandb.ai/links/shijiayang/13ckenoc

I first tried to tune the LR and increase the batch size, which does not bring much improvements. Then, I doubled number of layers from 4 to 8. I also add the LR scheduler with 600 step warm up and the rest of cosine scheduling, which enable me a larger max learning rate. I changed activation from GELU to SiLU, which does brings a minimal 0.1 drop in loss.

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
